<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dice Roller</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-games"/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelDie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Die:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDie">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>58</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>32</height>
     </size>
    </property>
    <property name="currentText">
     <string>6</string>
    </property>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelRoll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbNum">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>52</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelKeep">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbKeep">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>52</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Add:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbAdd">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>0</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>-2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelMin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbMin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>52</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelMax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbMax">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>52</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="value">
     <number>6</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRoll">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>250</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll</string>
    </property>
    <property name="shortcut">
     <string>Space</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtResult">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>191</width>
      <height>261</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtTotal">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>387</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
